### PR TITLE
fix: set card images to `object-fill` on desktop

### PR DESCRIPTION
**Pull request by DragosIorgulescu**
## Changes

Note: 
- setting a max height that maintains 1:1 aspect ratio messes up with the `fill` setting, so we would need to set `cover` or `fill` depending on the image height, is this something we want to do? Or maybe someone has other suggestions?

----

Screenshot 2024-01-18 at 15 11 03

----

Screenshot 2024-01-18 at 15 10 47

----

Screenshot 2024-01-18 at 15 10 38

## Events

n / a

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/cards/v1/WelcomePostCard.tsx [hunk 1]
```diff
@@ -125,7 +125,7 @@ export const WelcomePostCard = forwardRef(function SharePostCard(
               alt="Post Cover image"
               src={image}
               fallbackSrc={cloudinary.post.imageCoverPlaceholder}
-              className="my-2 object-cover"
+              className="my-2 object-cover mobileXXL:self-start mobileXXL:object-fill"
```
Comment (nensidosari) [inline packages/shared/src/components/cards/v1/WelcomePostCard.tsx:128]: Just suggestion to add this line in the component if it's being added to all components 🙏

**Review by nensidosari — APPROVED** [review]
LGTM, one non blocking comment.

Also 🙏 can you please post the note in the web channel, maybe someone has any ideas but just hasn't seen the PRs

File: packages/shared/src/components/cards/v1/WelcomePostCard.tsx (hunk 1 above)
Comment (DragosIorgulescu) [reply packages/shared/src/components/cards/v1/WelcomePostCard.tsx:128]: sry, missed your review. That's a good point, but these cards are used for the `share` and `feedback` cards as well, and we don't want these classes there 🙏

**Review by nensidosari — CHANGES_REQUESTED** [review]
This PR does not fix issues with large images as it doesn't allow for max height: https://dailydotdev.slack.com/archives/C02E2C3C13R/p1705657476639799?thread_ts=1705590446.763629&cid=C02E2C3C13R

Will come back to this.

**Review by nensidosari — APPROVED** [review]
This is good to go, large images to be tackled in another PR

**Pull request merged**